test/fail_compilation/fail19890a.d

```diff
@@ -1,7 +1,9 @@
-// PERMUTE_ARGS:
 /*
+PERMUTE_ARGS:
+TEST_OUTPUT:
 ---
-fail_compilation/fail19890a.d(8): Error: `void[/^[0-9]+(LU)?$/]` size 1 * /^[0-9]+$/ exceeds 0x7fffffff size limit for static array
+fail_compilation/fail19890a.d(9): Error: `void[$n$]` size 1 * $n$ exceeds 0x7fffffff size limit for static array
```

Comment:
There is no mention of this change in the description or title.

Reply:
This comment was just an informal description of the error message and not checked at all. The $n$ is necessary because the number depends on the test platform